client/rest/src/main/java/org/elasticsearch/client/Response.java

```diff
@@ -22,26 +22,25 @@
 import org.apache.http.Header;
 import org.apache.http.HttpEntity;
 import org.apache.http.HttpHost;
+import org.apache.http.HttpResponse;
 import org.apache.http.RequestLine;
 import org.apache.http.StatusLine;
 import org.apache.http.client.methods.CloseableHttpResponse;
 
-import java.io.Closeable;
-import java.io.IOException;
 import java.util.Objects;
 
 /**
  * Holds an elasticsearch response. It wraps the {@link CloseableHttpResponse} response and associates it with
  * its corresponding {@link RequestLine} and {@link HttpHost}.
  * It must be closed to free any resource held by it, as well as the corresponding connection in the connection pool.
```

Comment:
This comment is no longer true.

I wonder how big a loss this is. I imagine the point of having `CloseableHttpResponse` is so that you can read the entity directly from the underlying stream. Is that not possible with the async httpclient?